### ISSUE TITLE
Trained Video Swin Model

### ISSUE THREAD

**Issue by AshwinRamachandran2002** [KPeng9510/TransDARC#1]
Hi, 
Could you provide the configuration details of the video swin model used in your work?

**Comment by KPeng9510** [comment]
Hi, I will update the readme regarding it in these days, thank you for your question!

**Comment by KPeng9510** [comment]
Please check the readme for corresonding details, thank you!

**Comment by AshwinRamachandran2002** [comment]
Would it be possible to provide the trained model weights on the video swin baseline?

**Comment by KPeng9510** [comment]
I will consider to upload the model weights or the extracted features recently, if there is any update I will inform you in this issue

**Comment by KPeng9510** [comment]
The logits are uploaded, thank you!

**Comment by AshwinRamachandran2002** [comment]
Thank you! Please do consider uploading the model weights too.

**Comment by KPeng9510** [comment]
If there is no other question, this issue will be closed.
